crates/bevy_ecs/src/system/query.rs

```diff
@@ -1667,3 +1667,32 @@ impl<'w, D: QueryData, F: QueryFilter> Single<'w, D, F> {
         self.item
     }
 }
+
+/// [System parameter] that works very much like [`Query`] except it always contains at least one matching entity.
```

Comment:
Is this also applicable to `Single`/`Query::get_single`?
The only real difference is usage of `is_none` or `is_some`

I don't see this warning in `Query::get_single` but I can add it

Reply:
It doesn't feel as necessary there to me.  I wouldn't expect people to use `Single<T, Changed<U>>`, because it would be odd to react only if exactly one entity changes.  But I would expect people to use `Populated<T, Changed<U>>`, thinking they're getting better performance by skipping the system if nothing changes, while instead they're getting worse performance by doing the iteration in the executor.